<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsDialog</class>
 <widget class="QDialog" name="OptionsDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>130</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Опции</string>
  </property>
  <property name="windowOpacity">
   <double>0.800000000000000</double>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>371</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>10</y>
     <width>156</width>
     <height>74</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetMaximumSize</enum>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSpinBox" name="timeEdit">
      <property name="suffix">
       <string> сек.</string>
      </property>
      <property name="prefix">
       <string>Время: </string>
      </property>
      <property name="maximum">
       <number>10000000</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="frequencyEdit">
      <property name="prefix">
       <string>Частота: </string>
      </property>
      <property name="suffix">
       <string> КГц</string>
      </property>
      <property name="minimum">
       <double>0.100000000000000</double>
      </property>
      <property name="maximum">
       <double>1000000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="bandEdit">
      <property name="suffix">
       <string> Гц</string>
      </property>
      <property name="prefix">
       <string>Полоса: </string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10000000</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>135</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="addressEdit">
      <property name="placeholderText">
       <string>IP-адрес</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OptionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OptionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
